File: sqlparser/src/sql/csgen/SqlKillStatement.cs
using System;
using System.Collections;
using System.IO;
using System.Threading;
using System.Net;
using System.Net.Sockets;
using System.Text;
using RSSBus.core.j2cs;



using RSSBus.core;
using RSSBus;
namespace CData.Sql {


public class SqlKillStatement : SqlStatement {
  private string _queryId;

  public SqlKillStatement(Dialect dialectProcessor) : base(dialectProcessor) {
  }

  public string GetQueryId() {
    return _queryId;
  }

  public override void Accept(ISqlQueryVisitor visitor) {
    ;
  }

  public SqlKillStatement(SqlTokenizer tokenizer, Dialect dialectProcessor) : base(dialectProcessor) {
    tokenizer.NextToken();
    tokenizer.EnsureNextToken("QUERY");
    this._queryId = tokenizer.NextToken().Value;
  }

  public override Object Clone() {
    SqlKillStatement obj = new SqlKillStatement(null);
    obj.Copy(this);
    return obj;
  }

  protected override void Copy(SqlStatement obj) {
    base.Copy(obj);
    SqlKillStatement o = (SqlKillStatement)obj;
    this._queryId = o._queryId;
  }
}
}

File: sqlparser/src/sql/SqlKillStatement.java
//@
package cdata.sql;
//@
/*#
using RSSBus.core;
using RSSBus;
namespace CData.Sql {
#*/

import core.SqlExceptions;

public class SqlKillStatement extends SqlStatement {
  private String _queryId;

  public SqlKillStatement(Dialect dialectProcessor) {
    super(dialectProcessor);
  }

  public String getQueryId() {
    return _queryId;
  }

  public /*#override#*/ void accept(ISqlQueryVisitor visitor) throws Exception {
    ;
  }

  public SqlKillStatement(SqlTokenizer tokenizer, Dialect dialectProcessor) throws Exception{
    super(dialectProcessor);
    tokenizer.NextToken();
    tokenizer.EnsureNextToken("QUERY");
    this._queryId = tokenizer.NextToken().Value;
  }

  @Override
  public Object clone() {
    SqlKillStatement obj = new SqlKillStatement(null);
    obj.copy(this);
    return obj;
  }

  @Override
  protected void copy(SqlStatement obj) {
    super.copy(obj);
    SqlKillStatement o = (SqlKillStatement)obj;
    this._queryId = o._queryId;
  }
}

